docs/README.md

```diff
@@ -1,28 +1,116 @@
-# README
+# Documentation for Unity Catalog
 
-## Serving the documentation with mkdocs
+Unity Catalog uses [**MkDocs**](https://www.mkdocs.org/) to manage and serve its documentation. MkDocs is a simple, static site generator that’s geared towards project documentation and is written in Python. This guide will help you set up the local environment, serve the documentation locally, and deploy it to GitHub Pages.
 
-Create a virtual environment:
+## Setting Up MkDocs for Local Development
 
-```sh
+To start testing or modifying the documentation locally, follow these steps:
+
+### 1. Install MkDocs
```

Comment:
Not sure if this step is really needed, as MkDocs gets installed via the `requirements-docs.txt` file in step 3 (ideally in a virtual environment). Currently it makes the impression that you need to install as a first step. Alternatively, the usage via pipx could be introduced or it gets pointed out that the installation also happens in step 3.